cmd/litcli/helpers.go

```diff
@@ -0,0 +1,101 @@
+package main
+
+import (
+	"crypto/rand"
+	"encoding/binary"
+	"fmt"
+	"strconv"
+
+	"github.com/lightninglabs/lightning-terminal/session"
+	"github.com/urfave/cli"
+)
+
+// helperCommands are commands that do not require a connection to LiTd to
+// return a response.
+var helperCommands = cli.Command{
+	Name:     "helper",
+	Usage:    "helper commands",
+	Category: "Helper",
+	Subcommands: []cli.Command{
+		generateSuperMacRootIDCmd,
+		isSuperMacaroonCmd,
+	},
+}
+
+var generateSuperMacRootIDCmd = cli.Command{
+	Name: "supermacrootkey",
+	Usage: "Generate a valid super macaroon root key ID from scratch " +
+		"or from a given root key ID suffix",
+	Description: `
+	This command can be used to generate a valid super macaroon root key ID
+	from scratch or from a given root key ID suffix.
+	`,
+	Action: superMacRootKey,
+	Flags: []cli.Flag{
+		cli.StringFlag{
+			Name: "suffix",
+			Usage: "The suffix to use in the construction of " +
+				"the root key ID. If not provided, then a " +
+				"random one will be generated.",
+		},
+	},
+}
+
+func superMacRootKey(ctx *cli.Context) error {
+	var suffix [4]byte
+
+	if !ctx.IsSet("suffix") {
+		_, err := rand.Read(suffix[:])
+		if err != nil {
+			return err
+		}
+	} else {
+		suffixStr := ctx.String("suffix")
+		suffixInt, err := strconv.ParseUint(suffixStr, 10, 32)
+		if err != nil {
+			return err
+		}
+
+		binary.BigEndian.PutUint32(suffix[:], uint32(suffixInt))
+	}
+
+	id := session.NewSuperMacaroonRootKeyID(suffix)
+
+	fmt.Println("Root Key ID: ", id)
```

Comment:
It looks like for every other command (except the shutdown command) we return JSON. Should we do that here as well, just to be consistent? Can be an anonymous struct with JSON tags passed into the `printJSON()` function.

Reply:
good idea!